### PR TITLE
Added `regl.stats`

### PR DESCRIPTION
**Pull request by Erkaman**
Added `regl.stats` in  `lib/stats.js`. It currently only keeps track of this info(these counters are global, so they take all draw calls into account ):

```
 {
    bufferCount: 0,
    elementsCount: 0,
    framebufferCount: 0,
    shaderCount: 0,
    textureCount: 0,
    cubeCount: 0,
    renderbufferCount: 0
  }
```

I have modified the code so that these are correctly updated, and added tests in `test/stats.js` to verify that it is correctly implemented. 

Note that the test `stats.elementsCount==0 after regl.destroy()` will not pass due to issue #40. 

I was not sure whether I should put all these tests in their own file(which I am doing now), or whether all the tests for `bufferCount` should be put in `test/buffer.js`, and so on. What do you think?

### CODE REVIEW

**Comment by Erkaman** [comment]
If I run the tests locally with `npm run test`, then there is no problems. But circleci won't accept it for some reason.

**Comment by mikolalysenko** [comment]
Yeah, circleCI uses an old version of node so you can't use backtick string literals.  In your test cases you should just split up the string constants and use `Array.join`

File: test/stats.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    //
+    // Begin Test stats.shaderCount
+    //
+    regl = createREGL(gl)
+    stats = regl.stats
+
+    t.equals(stats.shaderCount, 0, 'stats.shaderCount==0 at start')
+
+    var draw1 = regl({
+      frag: `
```
Comment (mikolalysenko) [inline test/stats.js]: Use `Array.join()` instead of backticks for compatibility with old versions of node.

**Comment by Erkaman** [comment]
alright, I'll fix it

**Comment by Erkaman** [comment]
I had to comment out the failing test. Will uncomment it once issue #40 is resolved

**Comment by Erkaman** [comment]
Do you see any problems with the code, or can I merge?

**Comment by mikolalysenko** [comment]
I think we should fix issue #40 first, then merge

**Comment by Erkaman** [comment]
All tests now pass.

**Pull request merged**